deploy/gke/collector.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: collector
+  namespace: open-saves-namespace
+spec: 
+  schedule: "10 0 * * *"
```

Comment:
If I'm reading this properly, this reads as "run on 12:10am" every day. Is this your intent?